zio/zio.go

```diff
@@ -23,6 +24,7 @@ type WriterOpts struct {
 	EpochDates bool
 	Text       textio.WriterOpts
 	Zng        zngio.WriterOpts
+	ZSON       zsonio.WriterOpts
```

Comment:
Nit: We should be consistent about capitalization of acronyms, either by changing this to Zson or (in another PR) changing Zng and Zst to ZNG and ZST.

Reply:
I would vote for the latter.